stdlib/assoc.mc

```diff
@@ -103,11 +103,20 @@ let assocValues : AssocTraits k -> AssocMap k v -> [v] =
   lam _. lam m.
     map (lam t. t.1) m
 
+-- 'assoc2seq traits m' returns a sequence of all pairs stored in 'm'
+let assoc2seq : AssocTraits k -> AssocMap k v -> [(k, v)] =
+    lam _. lam m. m
+
```

Comment:
I can't see any difference compared to the existing `assoc2seq`? Additionally, the old version has not been removed?

Reply:
I didn't actually see the old one, oops